src/optimussettings.h

```diff
@@ -115,26 +115,26 @@ class OptimusSettings : public QSettings
     void setExternalPowerStartupMode(GPU gpu);
     static GPU defaultExternalPowerStartupMode();
 
-    // Intel
-    Driver intelDriver() const;
-    void setIntelDriver(Driver driver);
-    static Driver defaultIntelDriver();
+    // Integrated
+    Driver integratedDriver() const;
+    void setIntegratedDriver(Driver driver);
+    static Driver defaultIntegratedDriver();
```

Comment:
If I understand correctly, AMD, like Intel, has separate settings. So, we need to add separate settings for AMD. I think we should just add the Intel and AMD prefixes to the enums.